src/test/unit_tests/test_s3_ops.js

```diff
@@ -5,16 +5,15 @@ const _ = require('lodash');
 // setup coretest first to prepare the env
 const coretest = require('./coretest');
 coretest.setup({ pools_to_create: coretest.POOL_LIST });
-
+const config = require('../../../config');
 const AWS = require('aws-sdk');
 const http = require('http');
 const mocha = require('mocha');
 const assert = require('assert');
 const querystring = require('querystring');
-const SKIP_TEST = !process.env.AWS_ACCESS_KEY_ID || !process.env.AWS_SECRET_ACCESS_KEY;
+const USE_LOCAL_ENDPOINT = !process.env.AWS_ACCESS_KEY_ID || !process.env.AWS_SECRET_ACCESS_KEY ||
```

Comment:
```suggestion
const USE_LOCAL_ENDPOINT = process.env.USE_LOCAL_ENDPOINT === 'true';
```

Reply:
@guymguym I reverted the changes back to the original state and added some comments and logging.
The issue if we use`const USE_LOCAL_ENDPOINT = process.env.USE_LOCAL_ENDPOINT === 'true';` then I have to define default behavior in case neither the AWS endpoint nor USE_LOCAL_ENDPOINT is defined.

We will have to skip the `namespace bucket` add `namespace cache bucket` tests if neither variable is defined. This will make the test more complicated. We should just run all tests as part of travis/local testing. 

Please let me know if this is acceptable, or I will add a default behavior: no variable defined means skip the `namespace` related s3 tests.